caom2-soda-server/src/main/java/ca/nrc/cadc/caom2/soda/SodaJobRunner.java

```diff
@@ -364,7 +370,7 @@ void doit()
                                 {
                                     log.error("unexpected", ex);
                                     StringBuilder sb = new StringBuilder();
-                                    sb.append("Error: ").append(id).append(" vs");
+                                    sb.append(ex.getClass().getSimpleName() + ": ").append(id).append(" vs");
```

Comment:
Now that I see that the message in the UnsupportedOperationException matters, this should probably be:

"Error: " + ex.getMessage() + remaining stuff. 

And when you see use of append().append()... you shouldn't mix that style with String + operator. You can split the sequence of appends up into two lines (one for the error message parts and one for the specifics (append(id) and stuff tat follows.